docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,75 +4,372 @@ title: "Red Hat OpenShift"
+- [kubectl (1.30+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- [jq (1.7+)](https://jqlang.github.io/jq/download/) to interact with some variables.
+- [GNU envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html) to generate manifests.
+- [oc (version supported by your OpenShift)](https://docs.openshift.com/container-platform/4.17/cli_reference/openshift_cli/getting-started-cli.html) to interact with OpenShift.
+- A namespace to host the Camunda Platform, in this guide we will reference `camunda` as the target namespace.
 
-<Tabs queryString="current-scc">
-<TabItem value="w-scc" label="Security Context Constraints (SCCs)" default>
+## Deploy Camunda 8 via Helm charts
+
+### Configure your deployment
+
+Start by creating a `values.yml` file to store the configuration for your environment.
+This file will contain key-value pairs that will be substituted using `envsubst`.
+Over this guide, you will add and merge values in this file to configure your deployment to fit your needs.
+
+You can find a reference example of this file here:
+
+```hcl reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-ra-standard/aws/rosa-hcp/camunda-versions/8.7/procedure/install/helm-values/base.yml
+```
+
+:::warning Merging YAML files
+
+This guide references multiple configuration files that need to be merged into a single YAML file. Be cautious to avoid duplicate keys when merging the files. Additionally, pay close attention when copying and pasting YAML content. Ensure that the separator notation `---` does not inadvertently split the configuration into multiple documents.
+
+We strongly recommend double-checking your YAML file before applying it. You can use tools like [yamllint.com](https://www.yamllint.com/) or the [YAML Lint CLI](https://github.com/adrienverge/yamllint) if you prefer not to share your information online.
+
+:::
+
+#### Configuring the Ingress
+
+Before exposing services outside the cluster, we need an Ingress component. Here's how you can configure it:
+
+<Tabs queryString="current-ingress">
+
+<TabItem value="openshift-routes" label="Using OpenShift Routes" default>
+
+[Routes](https://docs.openshift.com/container-platform/latest/networking/routes/route-configuration.html) expose services externally by linking a URL to a service within the cluster. OpenShift supports both the [standard Kubernetes Ingress and routes](https://docs.openshift.com/container-platform/latest/networking/ingress-operator.html), giving cluster users the flexibility to choose.
+
+The presence of routes is rooted in their specification predating Ingress. The functionality of routes differs from Ingress; for example, unlike Ingress, routes don't allow multiple services to be linked to a single route or the use of paths.
+
+To use these routes for the Zeebe Gateway, configure this through Ingress as well.
+
+#### Prerequisite
```

Comment:
It might be worth removing this "Prerequisites" heading and allowing the subsequent H5s to be H4s. This nested, we lose the ToC that the additional header could add, and it allows the subsequent headers to stand out a little more.